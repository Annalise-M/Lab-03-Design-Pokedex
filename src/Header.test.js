// import React from 'react';
// import { shallow } from 'enzyme';
// import Header from './Header.js';

// exports[`renders the drop down for type 1`] = `
// <header>
//     <h1>
//         Welcome!!
//     </h1>
// </header>
// `;
