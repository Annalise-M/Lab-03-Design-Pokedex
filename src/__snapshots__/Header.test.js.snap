// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should welcome with greeting 1`] = `ShallowWrapper {}`;
